src/concern/mod.rs

```diff
@@ -21,10 +21,15 @@ use crate::{
 /// See the documentation [here](https://docs.mongodb.com/manual/reference/read-concern/) for more
 /// information about read concerns.
 #[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
 #[non_exhaustive]
 pub struct ReadConcern {
     /// The level of the read concern.
     pub level: ReadConcernLevel,
+
+    /// The snapshot read timestamp.
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
The server rejects commands with `atClusterTime: null`.

Reply:
Out of curiosity, would adding `#[serde(skip_serializing_none)]` to the `ReadConcern` struct do the trick here as well? We use that for a lot of our options structs but I'm wondering if that would break other behavior here.

Reply:
Ah, I'd missed that, thank you!  It looks like that works as well, updated to be consistent with the rest of the codebase.